site/app/views/grading/SimpleGraderView.php

```diff
@@ -18,30 +18,47 @@ class SimpleGraderView extends AbstractView {
     public function checkpointForm($gradeable, $rows, $graders) {
         $return = <<<HTML
 <div class="content">
+            <div style="float: right; margin-bottom: 10px; margin-left: 20px">
```

Comment:
Unindent as it it should be just one tab (4 spaces). The HTML and PHP content follow different indentation schemes (bit confusing at first, but you get used to it) which makes it easy to see where things start/end for various PHP blocks and HTML blocks.